angular15-vue3/vue-remote/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "vue-remote",
```

Comment:
Change name to `angular15-vue3_vue-remote`

Reply:
resolved